### PR TITLE
export tokenizer parameter

### PR DESCRIPTION
**Pull request by yuwen-yan**
Add back the tokenizer parameter, since it's hard to set tokenizer now.

### CODE REVIEW

File: tensorflow/models/rnn/translate/data_utils.py [hunk 1]
```diff
@@ -241,7 +241,7 @@ def data_to_token_ids(data_path, target_path, vocabulary_path,
           tokens_file.write(" ".join([str(tok) for tok in token_ids]) + "\n")
 
 
-def prepare_wmt_data(data_dir, en_vocabulary_size, fr_vocabulary_size):
+def prepare_wmt_data(data_dir, en_vocabulary_size, fr_vocabulary_size, tokenizer=None):
```
Comment (vrv) [inline tensorflow/models/rnn/translate/data_utils.py]: Probably worth adding documentation in the doc string for this.

**Comment by vrv** [comment]
Assigning to @ludimagister for general API

File: tensorflow/models/rnn/translate/data_utils.py (hunk 1 above)
Comment (ybbaigo) [inline tensorflow/models/rnn/translate/data_utils.py]: done, thanks for your reminder

**Comment by ludimagister** [comment]
Yes, this should be fine. Thanks for adding.

**Comment by vrv** [comment]
Can you squash the commits?  Then we'll test and merge

**Comment by ybbaigo** [comment]
@vrv sure, done.

**Comment by vrv** [comment]
Thanks!

@tensorflow-jenkins: test this please

**Comment by ybbaigo** [comment]
@vrv I just checked the log, looks weird, what should I do now?
> //tensorflow/user_ops:ackermann_test                                     PASSED in 1.9s
> //tensorflow/python:sparse_add_op_test                                  TIMEOUT in 65.0s
>   /var/lib/jenkins/workspace/tensorflow-pull-requests-cpu/bazel-ci_build-cache/.cache/bazel/_bazel_jenkins/eab0d61a99b6696edb3d2aff87b585e8/workspace/bazel-out/local_linux-fastbuild/testlogs/tensorflow/python/sparse_add_op_test/test.log
> 
> Executed 222 out of 387 tests: 386 tests pass and 1 fails locally.
> 
> Parameterized build ends with FAILURE at: Wed Mar 30 01:29:11 UTC 2016 (Elapsed time: 378 s)
> Build step 'Execute shell' marked build as failure
> Unable to get pull request builder trigger!!
> Setting status of 19083e1e2f97bb46ecfff9fad35c7f4d10cdc2cb to FAILURE with url http://ci.tensorflow.org/job/tensorflow-pull-requests-cpu/525/ and message: 'FAILURE
>  '
> Using context: Linux CPU Tests
> Finished: FAILURE

**Comment by vrv** [comment]
I think it's a 'flaky test' due to timeouts -- I think it's fine, so I'm going to merge.

**Pull request merged**